README.md

```diff
@@ -56,29 +56,34 @@ For pre-release versions, such as `v1.11.0-rc.0`, use the full version
 specifier (`v1.11.0-rc.0`) and set option `version-type` to `strict`. Pre-release versions are
 opt-in, so `1.11.x` will not match a pre-release.
 
-#### "Latest" versions
+#### "latest" versions
 
-Set a tool's version to `latest` to retrieve the latest version of a given tool.
+Set a tool's version to `latest` to retrieve the latest non-RC version of a given tool.
 The latest version is (locally) calculated by the action based on the (retrieved) versions
 it knows (**note**: it is not the same as [GitHub considers it](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository)
 and some repositories might propose).
 
 If in doubt do a test run and compare the obtained release with the one you were expecting to
 be the latest.
 
+#### "latest-rc" versions
+
+Similar to `latest`, except that release candidates are also considered.
+To target these, set a tool's version to `latest-rc`.
+
 ### Compatibility between Operating System and Erlang/OTP
 
 This list presents the known working version combos between the target operating system
 and Erlang/OTP.
 
-| Operating system | Erlang/OTP   | OTP Architecture | Status
-|-                 |-             | -                |-
```

Comment:
I think the only broken thing was the space after `|` below "OTP Architecture". I'd like to keep as is, instead of aligning the code visually since, e.g. a future change might imply changing multiple lines, which won't be good.